src/main/java/edu/harvard/iq/dataverse/util/BundleUtil.java

```diff
@@ -48,11 +46,30 @@ public static String getStringFromBundle(String key, List<String> arguments, Res
             return stringFromBundle;
         }
     }
+    
+    //This call was added to allow bypassing the exception catch, for filetype indexing which also catches it
+    public static String getStringFromBundle(String key, List<String> arguments, ResourceBundle bundle) {
+        try {
+            return getStringFromBundleNoMissingCheck(key, arguments, bundle);
+        } catch (MissingResourceException ex) {
+            logger.warning("Could not find key \"" + key + "\" in bundle file: ");
+            logger.log(Level.CONFIG, ex.getMessage(), ex);
+            return null;
+        }
+    }
 
     public static String getStringFromPropertyFile(String key, String propertyFileName  ) {
         ResourceBundle bundle = getResourceBundle(propertyFileName);
         return getStringFromBundle(key, null, bundle);
     }
+    
+    //Added this function to fix indexing issue where it was expecting the exception
+    // to bubble up. See FileUtil.getFacetFileType --MAD 4.9.4
+    public static String getStringFromPropertyFileAllowMissing(String key, String propertyFileName  ) {
```

Comment:
Can this comment be converted to javadoc?

Reply:
In general, I feel like the change could be a focused a little more on how getStringFromPropertyFile works. That method is now only used twice? Instead of having it and getStringFromPropertyFileAllowMissing can we just have a single method?

Before (develop):

![screen shot 2018-10-22 at 10 52 35 am](https://user-images.githubusercontent.com/21006/47299933-55e57f00-d5e9-11e8-9bc6-9f53003bd1c5.png)


After (this pull request):

![screen shot 2018-10-22 at 10 53 11 am](https://user-images.githubusercontent.com/21006/47299941-5a119c80-d5e9-11e8-9ddd-b07f78b35750.png)

Reply:
After the latest commit!

![screen shot 2018-10-22 at 11 43 50 am](https://user-images.githubusercontent.com/8901015/47304088-a235bc80-d5f3-11e8-9ad7-ccdece7ccc6c.png)

Thanks for the feedback, its even better now

Reply:
I decided to switch getResourceBundle back to public so it can be used in the future alongsize getStringFromBundle passing in an arbitrary bundle, even though right now it doesn't need the scope. Allows better testing as well.